src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -0,0 +1,57 @@
+@import "vanilla-framework/scss/settings_colors";
+
+.p-primary-nav {
+  a,
+  a:hover {
+    color: $color-light;
+  }
+
+  &__header {
+    display: grid;
+    grid-template-columns: 1fr 40px;
+    margin-bottom: -1rem;
+    padding: 0.5rem;
+  }
+
+  &__toggle {
+    background: none;
+    border: 0;
+    transition: transform 0.25s ease-in-out;
+
+    &:hover {
+      background: 0;
```

Comment:
This should be `background: none` but why do you need it if the toggle already had it above?

Reply:
Vanilla gives button hover a specific grey background which is not overridden by the toggle above.